docs/tutorials/platform_beginner/7_embed_content.md

```diff
@@ -62,36 +62,36 @@ Add the template for the line view of a Landmark by creating `templates/line/lan
 
 ``` html+twig hl_lines="4"
 <section>
-<div class="col-xs-4 photos-box">
-    <a href="#bikeModal{{ content.id }}" data-toggle="modal">
-        {{ ez_render_field( content, 'photo', { parameters: { 'alias': 'landmark_list', 'class': 'img-responsive img-rounded'}}) }}
-    </a>
-</div>
-
-{# MODAL #}
-<div class="bike-modal modal fade" id="bikeModal{{ content.id }}" tabindex="-1" role="dialog" aria-hidden="true">
-    <div class="modal-content">
-        <div class="close-modal" data-dismiss="modal">
-            <div class="lr">
-                <div class="rl">
+    <div class="col-xs-4 photos-box">
+        <a href="#bikeModal{{ content.id }}" data-toggle="modal">
+            {{ ez_render_field( content, 'photo', { parameters: { 'alias': 'landmark_list', 'class': 'img-rounded'}}) }}
+        </a>
+    </div>
+
+    {# MODAL #}
+    <div class="bike-modal modal fade" id="bikeModal{{ content.id }}" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-content">
+            <div class="close-modal" data-dismiss="modal">
+                <div class="lr">
+                    <div class="rl">
```

Comment:
Those `divs` with class `rl` and `lr` looks strange. What is the purpose of having it?

Reply:
They are responsible for creating the "X" from the exit button from the module: 
![image](https://user-images.githubusercontent.com/13621049/73746956-a63d0d80-4756-11ea-87ed-085cdc1e770e.png)
There's probably  more elegant way to do it?